main.js

```diff
@@ -1 +1,15 @@
 
+function dis (value){
+document.getElementById("edu").value+=value;
+}
+
+function solve(){
+let x = document.getElementById("edu").value;
+let y = eval(x);
+document.getElementById("edu").value = y;
+}
+
+function clr(){
+document.getElementById("edu").value = "";
+
+}
```

Comment:
love the effort on this, Clayton! Love that you're working to implement the functions in JavaScript. Keep pushing with the DOM manipulation!